app/keymap-module/modules/modules.cmake

```diff
@@ -107,6 +114,21 @@ foreach(root ${BOARD_ROOT})
     endif()
 endforeach()
 
+if(EXISTS ${BOARD_DIR}/revision.cmake)
+    # Board provides revision handling.
+    include(${BOARD_DIR}/revision.cmake)
+elseif(BOARD_REVISION)
+    message(WARNING "Board revision ${BOARD_REVISION} specified for ${BOARD}, \
+                     but board has no revision so revision will be ignored.")
+endif()
+
+if(DEFINED BOARD_REVISION)
+    string(REPLACE "." "_" BOARD_REVISION_STRING ${BOARD_REVISION})
+    set(KEYMAP_BOARD_REVESION_PREFIX "${BOARD}_${BOARD_REVISION_STRING}")
```

Comment:
Typo (and below) 😅 
```suggestion
    set(KEYMAP_BOARD_REVISION_PREFIX "${BOARD}_${BOARD_REVISION_STRING}")
```

Reply:
Oh man, did this in my commit message and thought "Man, glad I caught that!".

Thanks.